pkg/mev/base_chain_sender.go

```diff
@@ -0,0 +1,73 @@
+package mev
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+type BaseChainSender struct {
+	c          *http.Client
+	endpoint   string
+	senderType BundleSenderType
+}
```

Comment:
Maybe in the future we can use this struct for other L2, a common struct name will be better

Reply:
renamed to L2Sender